vendor/engines/split_accounts/spec/controllers/facility_accounts_controller_spec.rb

```diff
@@ -0,0 +1,126 @@
+require "rails_helper"
+
+RSpec.describe FacilityAccountsController do
+  render_views
+
+  before(:all) do
+    SplitAccounts::Engine.enable! unless SettingsHelper.feature_on?(:split_accounts)
+  end
+
+  after(:all) do
+    SplitAccounts::Engine.disable! unless SettingsHelper.feature_on?(:split_accounts)
+  end
+
+  let(:facility) { FactoryGirl.create(:setup_facility) }
+  before { sign_in user }
+
+  describe "as an admin" do
+    let(:user) { FactoryGirl.create(:user, :administrator) }
+
+    describe "default new" do
+      before { get :new, facility_id: facility.url_name, owner_user_id: user.id }
+
+      it "sees split account option" do
+        expect(response.code).to eq("200")
+        expect(response.body).to include("Chart String")
+        expect(response.body).to include("Split Account")
```

Comment:
After looking at it, it's kind of a pain, and also since this is effectively an integration test (we're rendering out the view, just skipping the routing), I think I'd like it to break if we change things.